app/login/page.tsx

```diff
@@ -19,8 +19,12 @@ import { loginSchema } from "@/lib/validation";
 import { Separator } from "@/components/ui/separator";
 import { signIn } from "next-auth/react";
 import { loginWelcome } from "@/constants";
+import { useTheme } from "../context/ThemeProvider";
 
 const Login = () => {
+  const { mode, setMode } = useTheme();
+  const colorsLogIn = ["bg-[#E7FAF4]", "bg-[#FDF4EA]"];
```

Comment:
Constants should in general be outside a component, otherwise they are re-initialised on every re-render.